packages/ethereum-contracts/test/test-solc-compatibility.sh

```diff
@@ -35,5 +35,10 @@ find contracts/interfaces/ -name '*.sol' | while read i;do
   $SOLC --allow-paths . $i
 done
 
+find contracts/apps/ -name '*.sol' | while read i;do
```

Comment:
There are some real compatibility issues, I bypassed/fixed it for now. Check it out.